tickets/src/__tests__/components/content/CreateContent.test.js

```diff
@@ -108,14 +114,12 @@ describe('CreateContent', () => {
             addEvent={addEvent}
             now={now}
             loadEvent={jest.fn()}
+            event={event}
           />
         </ConfigProvider>
       </Provider>
     )
 
-    rtl.fireEvent.change(form.getByTestId('Choose a lock'), {
```

Comment:
The selecting of the lock is now done inside another "sub-component" for clarity.